src/messages/GE.json

```diff
@@ -378,5 +378,10 @@
   "consent_required_consent": "Please consent to the permission to access this feature.",
   "interaction_required_consent": "Please wait for the consent process to finish and verify that pop-ups are enabled.",
   "user_cancelled_consent": "Please wait for the consent process to finish.",
-  "access_denied_consent": "Your consent to this permission has been blocked by your tenant admin. Ask your admin to grant you access and then try again."
-}
\ No newline at end of file
+  "access_denied_consent": "Your consent to this permission has been blocked by your tenant admin. Ask your admin to grant you access and then try again.",
+  "This response contains unviewable content": "This response contains content that may not be viewable on this editor.",
+  "Click here to download": "Click here to download.",
+  "File response is available in original format only": "The file response is only available in the original format due to limitations of the Javascipt SDK.",
+  "Response is result of workaround": "A workaround has been used to fetch the download URL by fetching the download URL using query, await client.api('/drive/items/{item-id}').get().select('content.downloadUrl') and using the returned URL to download file contents.",
```

Comment:
**suggestion**: The message can be reworded so that the code snippet is left outside of the translation text and put into the code at the very end of the message